### DISCUSSION TITLE
Release v1.12.9 / 5.67.9

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#3606] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

### Release Notes

This is a maintenance release, it fixes many issues, essentially a regression introduced in 1.12.8

For a full list of changes and fixes please review the [1.12.8](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#1128--5678---2024-01-31) and [1.12.9](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#1129--5679---2024-02-06) change logs.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.12.9'>Release v1.12.9 / 5.67.9</a>.</em>

**Comment by BilleBarrett** [comment]
I have under Run Sandboxed -Action 
Add 'Run Sandboxed' to the explorer context menu  " It does not appear" it use to
Run Unsandboxed runs OK
![sandboxiecontext](https://github.com/sandboxie-plus/Sandboxie/assets/41338917/97c7a8f2-5e21-4db5-81fd-633212117d04)

**Reply by isaak654** [reply]
Did you try a reboot?

**Comment by BloodySunrise** [comment]
![image](https://github.com/user-attachments/assets/ba8a8551-c0af-4e71-be89-972117d6baee)

Hello, when installing Sandboxie software version 5.67.9 on Windows 11, the following error appears

**Reply by littledoggi** [reply]
same problem
need only this version
